components/Feature/VulnerabilitiesGrid/index.js

```diff
@@ -32,7 +32,7 @@ function createLookup() {
   return lookup;
 }
 
-const VulnerabilitiesGrid = ({ resources, onUpdate, residentCoordinates, genericPostcode, onError , updateSelectedResources, customerId}) => {
+const VulnerabilitiesGrid = ({ resources, onUpdate, residentCoordinates, genericPostcode, onError , updateSelectedResources, customerId, prompts}) => {
```

Comment:
Were the changes to this file being used for testing? Is it good to remove them from this PR?

Reply:
ok good shout, I did use this for testing but ill remove them now